doc/api_methods/AddSubscriber.md

```diff
@@ -0,0 +1,44 @@
+[back to list](../Readme.md)
+
+# Add Subscriber
+
+## `array addSubscriber(array $subsriber [, array $list_ids = [], array $options = []])`
+
+This method allows to create a subscriber, add him into lists and handles confirmation email and admin notification email sending and welcome email scheduling.
+
+If sign-up confirmation is enabled in the MailPoet settings a subscriber is created with status `unconfirmed` otherwise the status is set to `subscribed`.
+
+*A confirmation email* is an email which is sent to a subscriber so that he can confirm his subscription. It is sent only if sign-up confirmation is enabled in the MailPoet settings.
+*A welcome email* is an automatic email which is sent to a new subscriber. This email is scheduled only if sign-up confirmation is disabled and some welcome email is configured for some of given lists.
+*An admin notification email* is sent to the site admin to inform him about a new subscription. It is sent only if the notification feature is enabled in the MailPoet setting.
+All these emails can be disabled using `$options`.
+
+A subscriber can be added only once. This method throws an `\Exception` in case of adding an existing subscriber.
+
+It returns a new subscriber. See [Get Subscriber](GetSubscriber.md) for a subscriber data structure.
+
+## Arguments
+### `$subsciber` (required)
+
+An associative array containing subscriber data which contains default properties (email, first_name, last_name) and custom subscriber fields which were defined in MailPoet.
+It has to contain an email and all required custom fields. To get defined custom fields see [Get Subscriber Fields](GetSubscriberFields.md)
+
+| Property | Type | Description |
+| --- | --- | --- |
+| email (required) | string | a valid email address |
+| first_name (optional) | string/null | Fist name of the subscriber. |
+| last_name (optional) | string/null |  Last name of the subscriber. |
+| cf_* (optional/required) | string/null | A custom field (see [Get Subscriber Fields](GetSubscriberFields.md)).  |
```

Comment:
Are you sure it is `string` here? I thought we allow to specify custom fields that are numbers.

Reply:
Yes. I double checked.